Formula/a/ace.rb

```diff
@@ -1,8 +1,8 @@
 class Ace < Formula
   desc "ADAPTIVE Communication Environment: OO network programming in C++"
   homepage "https://www.dre.vanderbilt.edu/~schmidt/ACE.html"
-  url "https://github.com/DOCGroup/ACE_TAO/releases/download/ACE%2BTAO-7_1_2/ACE+TAO-7.1.2.tar.bz2"
```

Comment:
I just revert the url change as I feel like that is more of github side issue

Reply:
Sure, thanks for digging. I thought I was hitting a blocker on `curl` not even being able to download it, but it's possible I was missing something!